app/admin/billing/contractors.rb

```diff
@@ -6,10 +6,25 @@
   acts_as_clone
   acts_as_safe_destroy
   acts_as_status
+  acts_as_async_destroy('Contractor')
+  boolean = [ ['Yes', 't'], ['No', 'f']]
```

Comment:
I propose(may no work) to move this variable to the method in ResourceDSL and use it instead.

```ruby
# lib/resource_dsl/acts_as_async_update.rb
module ResourceDSL
  class ActsAsAsyncUpdate
    def boolean_select_options
      [ ['Yes', 't'], ['No', 'f'] ]
    end
  end
end
```

Reply:
Nice idea. Works fine.